src/it/scala/org/broadinstitute/dsde/rawls/WorkspaceSimulation.scala

```diff
@@ -51,9 +48,14 @@ class WorkspaceSimulation extends FlatSpec with WorkspaceApiService with EntityA
   def httpJson[T](obj: T)(implicit writer: JsonWriter[T]) = HttpEntity(ContentTypes.`application/json`, obj.toJson.toString())
   def repeat[T](n: Int)(exp: => T) = (1 to n) map (_ => exp)
 
+  // get config, with fallback. conventionally the database config is stored in /etc, but in cases where /etc
+  // is not accessible (due to e.g. permissions issues) we can populate application.conf instead
+  val etcConf = ConfigFactory.parseFile(new File("/etc/rawls.conf"))
+  val appConf = ConfigFactory.parseFile(new File("application.conf"))
```

Comment:
Right now, I'm having Jenkins populate it, because writing to ```/etc/rawls.conf``` requires a bunch of extra permissions setting. But it would probably be better to call it ```jenkins.conf``` or something so as not to confuse it with the other ```application.conf``` files sitting in the repo.